### PR TITLE
EGI-ESG SLA news: Create index.md

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
News entry created about the SLA between EGI and EuroScienceGateway

### CODE REVIEW

File: content/news/2023-10-10-sla-esg-egi/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+The Service-Level Agreement (SLA) signed between EGI and ESG formalizes access to resources on INFN's Bari cloud and [IISAS](https://www.egi.eu/partner/iisas/)'s FedCloud, adding a Slovakian location for data processing to usegalaxy.eu's resource network. In summary, the available computing resources will be extended by 130 vCPUs, 470 GB RAM and 4 TB online storage. Beyond cloud compute and storage capacities, usegalaxy.eu's virtual organization (VO) will be supported with EGI's Workload Manager and Check-in services, allowing to monitor the allocation of resources that are used to carry out tasks in the ESG project.
```
Comment (bgruening) [inline content/news/2023-10-10-sla-esg-egi/index.md:16]: Not sure if we need to specify the resources in that detail.

File: content/news/2023-10-10-sla-esg-egi/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+hide_tease: false
+subsites: [all-eu, esg]
+main_subsite: eu
+tags: [eurosciencegateway, sla, egi, service level agreement]
+authors: Sebastian Schaaf
+authors_structured:
+- github: sebastian-schaaf
+---
+
+In the Galaxy-driven EOSC project [EuroScienceGateway (ESG)](https://www.eurosciencegateway.org) one of 18 partners is [EGI - the European Grid Initiative](https://www.egi.eu/about/) is a federation of computing and storage resource providers in Europe, facilitating up a distributed infrastructure. The 29 participants  from 22 countries are united by the mission to deliver advanced computing and data analytics services for research and innovation. They provide access to more than 1.2 Exabytes of research data and 1.2 Million CPU Cores for data processing. Notably, with [CESNET](https://www.egi.eu/partner/cesnet/) (CZ), [CNRS](https://www.egi.eu/partner/cnrs/) (FR), [CYFRONET](https://www.egi.eu/partner/cyfronet/) (PL), [INFN](https://www.egi.eu/partner/infn/) (IT) and [TUBITAK ULAKBIM](https://www.egi.eu/partner/tubitak-ulakbim/) (TR), five of these partners are partners of ESG, too.
```
[the remark this answers is not in the document]
Comment (sebastian-schaaf) [reply content/news/2023-10-10-sla-esg-egi/index.md]: True... rephrased.

File: content/news/2023-10-10-sla-esg-egi/index.md (hunk 1 above)
Comment (sebastian-schaaf) [reply content/news/2023-10-10-sla-esg-egi/index.md]: Me neither... removed.

**Comment by sebastian-schaaf** [comment]
Reviewed/edited article following the three comments above.

**Pull request merged**